File: back-office/book-api/src/main/java/com/moyu/book/dao/pojo/Status.java
package com.moyu.book.dao.pojo;

import java.io.Serializable;
import lombok.Data;

/**
 * 
 * @TableName status
 */
@Data
public class Status implements Serializable {
    /**
     * 
     */
    private Long id;

    /**
     * 座位表状态
     */
    private Boolean status;

    /**
     * 开始时间
     */
    private Long sdate;

    /**
     * 结束时间
     */
    private Long ddate;

    /**
     * 申请者id
     */
    private Long userId;

    /**
     *
     */
    private Long statusId;
    private static final long serialVersionUID = 1L;
}

File: android/app/src/main/java/com/github/book/entity/Status.kt
package com.github.book.entity

import com.google.gson.annotations.SerializedName

/**
 * description none
 * author ez_yang@qq.com
 * date 2021.11.25 上午 11:19
 * version 1.0
 * update none
 **/
data class Status(
    val id: String,
    val status: Boolean,
    @SerializedName("sdate")
    val sDate: Long,
    @SerializedName("ddate")
    val dDate: Long,
    val userId: String,
    val statusId: String
)
